src/elements/content-explorer/__tests__/ContentExplorer-test.js

```diff
@@ -118,32 +118,25 @@ describe('elements/content-explorer/ContentExplorer', () => {
             wrapper = getWrapper();
             instance = wrapper.instance();
             instance.api = { getFileAPI };
-            instance.setState = jest.fn();
             instance.closeModals = jest.fn();
             instance.updateCollection = jest.fn();
 
             return instance.fetchFolderSuccessCallback(collection, false).then(() => {
-                expect(instance.setState).toHaveBeenCalled();
                 expect(instance.closeModals).toHaveBeenCalled();
-                expect(instance.updateCollection).toHaveBeenCalledWith(collection, undefined);
+                expect(instance.updateCollection).toHaveBeenCalledWith(collection, undefined, expect.any(Function));
             });
         });
 
         test('thumbnail url should be assigned to item if grid view is enabled', () => {
             wrapper = getWrapper({ features: { contentExplorer: { gridView: { enabled: true } } } });
             instance = wrapper.instance();
             instance.api = { getFileAPI };
-            instance.setState = jest.fn();
             instance.closeModals = jest.fn();
             instance.updateCollection = jest.fn();
 
             return instance.fetchFolderSuccessCallback(collection, false).then(() => {
-                expect(instance.setState).toHaveBeenCalled();
                 expect(instance.closeModals).toHaveBeenCalled();
-                expect(instance.updateCollection).toHaveBeenCalledWith(
-                    { ...collection, items: [{ ...item, thumbnailUrl }] },
-                    undefined,
-                );
+                expect(instance.updateCollection).toHaveBeenCalledWith(collection, undefined, expect.any(Function));
```

Comment:
Will write tests to make sure `updateCollection` properly populates thumbnailUrl fields once approved.